contracts/access/Roles.sol

```diff
@@ -37,10 +37,12 @@ library Roles {
     function removeAll(Role storage role, address account) internal {
```

Comment:
Lets call this method `reset` because this implies that after call the function there is just one account with the role.